libs/sdk-ui-kit/src/Dialog/ConfirmDialogBase.tsx

```diff
@@ -0,0 +1,71 @@
+// (C) 2020 GoodData Corporation
+import React from "react";
+import cx from "classnames";
+import noop from "lodash/noop";
+import { Button } from "../Button";
+import { DialogBase } from "./DialogBase";
+import { IConfirmDialogBaseProps } from "./typings";
+
+/**
+ * @internal
+ */
+export class ConfirmDialogBase extends DialogBase<IConfirmDialogBaseProps> {
+    static defaultProps: Partial<IConfirmDialogBaseProps> = {
+        displayCloseButton: true,
+        onCancel: noop,
+        onSubmit: noop,
+        isSubmitDisabled: false,
+    };
+
+    constructor(props: IConfirmDialogBaseProps) {
+        super(props);
+
+        this.onKeyDown = this.onKeyDown.bind(this);
```

Comment:
rewrite to => fce to avoid bind

Reply:
I added the arrow function to the render method because it is not a performance heavy operation. 

To be honest, I am a little bit afraid to use the arrow function in a class property like this:

```
onKeyDown = (event: React.KeyboardEvent<HTMLDivElement>): void => {
        this.onKeyDown(event);
}
```

I am not sure how React will deal with this when we extend DialogBase and not React.Components.

Reply:
After discussion with @ivan-nejezchleb I have changed it to the arrow function in a class property.